activity_calendar/templates/activity_calendar/activity_page_base.html

```diff
@@ -0,0 +1,203 @@
+{% extends 'core/base.html' %}
+
+{% load static %}
+{% load get_login_url %}
+{% load calendar_tags %}
+
+{% block title %}
+    Squire - Calendar
+{% endblock title %}
+
+{% block css %}
+    {{ block.super }}
+    <link rel="stylesheet" href="{% static 'css/slot_frames.css' %}">
+{% endblock css %}
+
+{% block js %}
+    {{ block.super }}
+    {% if show_modal %}
+        <script type="text/javascript">
+            $(window).on('load', function () {
+                $('#slot-creation-modal').modal('show');
+            });
+        </script>
+    {% endif %}
+{% endblock js %}
+
+{% block content-frame-class %}
+    wideContentFrame
+{% endblock %}
+
+{% block content %}
+    {% if subscriptions_open and not user.is_authenticated %}
+        <div id="error-msg" class="alert alert-danger alert-dismissible fade show" role="alert">
+            <strong class="alert-danger">Oh noes!</strong> Registrations are currently closed!
+            <button type="button" class="close" data-dismiss="alert" aria-label="Close">
+                <span aria-hidden="true">&times;</span>
+            </button>
+        </div>
+    {% endif %}
+
+    <!-- Slots -->
+    <h1>{{ activity.title }}</h1>
+    <div class="row">
+        <div class="col-12 col-md-7">
+            {% block activity-info %}
+                <div class="d-flex">
+                    <!-- Calendar Icon -->
+                    <div class="">
+                        <img
+                                src="{% static 'octicons/calendar.svg' %}"
+                                alt="Calendar-icon"
+                                height="16"
+                                width="16"
+                        />
+                    </div>
+                    <!-- Event Date & Recurrence Information -->
+                    <div class="col align-self-center">
+                        <div class='nospace'>
+                            <p id="event-date">
+                                {% if start_date.date == end_date.date %}
+                                    <!-- Activity starts and ends on the same day -->
+                                    {{ start_date|date:"l, j E H:i" }} - {{ end_date|date:"H:i" }}
```

Comment:
1. REMARK: I've checked the manual and the used parameters seem sensible: https://docs.djangoproject.com/en/3.1/ref/templates/builtins/#date Maybe link these docs as the date formats aren't common across the code?
2. Why are you putting a comma between the weekday and day here, and between date and time on L65?